spotify_connect_scrobbler/lastfm.py

```diff
@@ -2,21 +2,17 @@
 import hashlib
 import requests
 
-from .credentials import LastfmCredentials
-
-
 class LastfmClient:
     """ A simple client for the Last.fm API."""
 
-    def __init__(self, key, secret):
+    def __init__(self, creds):
```

Comment:
If the webservice scrobbles for several users we probably don't want to pass the users' tokens to the client cause then we would have to create a client for each user. I'd say we try to make the clien as stateless as possible.